services/frontendhost/frontend.go

```diff
@@ -303,13 +303,38 @@ func convertCGZoneConfigToInternal(cgZoneCfg *c.ConsumerGroupZoneConfig) *shared
 
 // convertCreateCGRequestToInternal converts Cherami CreateConsumerGroupRequest to internal shared CreateConsumerGroupRequest
 func convertCreateCGRequestToInternal(createRequest *c.CreateConsumerGroupRequest) *shared.CreateConsumerGroupRequest {
+
+	// detect and correct the units for 'startFrom' (expected internally to be in nanoseconds)
+	startFrom := createRequest.GetStartFrom()
```

Comment:
Will do. Though I still think it is better to test the behaviour of "CreateConsumerGroup" itself .. not just the helper. In the future if, say, someone does away with calling the helper, the test (that only tests the helper) would still pass.